bitwise_operators/check_even_odd.cpp

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
as discussed in reviews above, this file should be removed from this pull-request